scalableminds/webknossos
# Introducing a per-layer scale for datasources

This PR implements the backend changes for #2295. I adjusted the frontend accordingly, so while these changes will not have any effect, they are save to merge into master, before finishing the frontend part. This is therefore ready for review imo.

### Steps to test:
- import / update a wkw dataset
- the presented json should contain a `scale` property for resolution in each in layer, that can be updated and is then delivered to the client on the `api/datasets/...` endpoint
- the backen furthermore checks, that 1) there is only one scale for each resolution across different layers (i.e. different layers cannot have different scales for the same resolution) 2) the scales are (not strictly) monotonically increasing in each dimension

------
